docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -194,60 +189,76 @@ Content-Type: application/json
 ```
 
+<Callout type="info">
+  * If your token were to become compromised and you wish to severe the Storefront API connection for your token, you can use the [Revoke a token](/docs/storefront-auth/tokens#revoke-a-token) endpoint. Only use this in emergency situations, and do not revoke tokens unnecessarily. Instead, use a shorter expiration and allow them to expire naturally.
+</Callout>
+
 ### Customer login
 
-If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the new Customer Login mutation to sign in to a customer account with an email address and a password. For server-side integrations, consider a customer impersonation token instead. This will set a session cookie in the browser which will authenticate the customer account on future requests:
+If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the Customer Login mutation to sign in to a customer account with an email address and a password. This will set a session cookie in the browser which will authenticate the customer account on future requests:
 
 ```graphql filename="Example query: login mutation" showLineNumbers copy
 mutation Login($email: String!, $pass: String!) {
   login(email: $email, password: $pass) {
     result
+    customer {
+      entityId
+      firstName
+      lastName
+      email
+    }
   }
 }
 ```
 
-As a best practice, you should inject the password using GraphQL query variables. This prevents the password from being exposed in the query itself. In the [GraphQL Playground](/graphql-playground), you can set the variables for the request.
+This mutation may also be useful for server-to-server applications using a Customer Impersonation Token, as you can use this interface to validate a customer email address + password, for example to power a login form. When the credentials are correct and the mutation returns successfully, you can take the resulting customer ID and store it in a session to use in the `x-bc-customer-id` header in future requests for that shopper.
+
+As a security best practice, you should inject the email address & password using GraphQL query variables. This prevents the password from being exposed in the query itself. In the [GraphQL Playground](/graphql-playground), you can set the variables for the request.
 
 ![GraphQL Playground Query Variables](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-overview-01.png "GraphQL Playground Query Variables")
 
 You can use a logout mutation to sign out of a customer account:
 
-
 ```graphql filename="Example query: logout mutation" showLineNumbers copy
 mutation Logout {
-   logout {
-     result
-   }
- }
+  logout {
+    result
+  }
+}
 ```
 
+This will clear the session cookie which has the effect of logging out the shopper, and future GraphQL requests will be from the perspective of an anonymous shopper.
```

Comment:
```suggestion
This will clear the session cookie, which has the effect of logging out the shopper, and future GraphQL requests will be from the perspective of an anonymous shopper.
```